moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -119,15 +119,41 @@ trajectory_msgs::msg::JointTrajectory composeTrajectoryMessage(const servo::Para
   joint_trajectory.header.frame_id = servo_params.planning_frame;
   joint_trajectory.joint_names = joint_state.joint_names;
 
-  trajectory_msgs::msg::JointTrajectoryPoint point;
+  static trajectory_msgs::msg::JointTrajectoryPoint point;
```

Comment:
Why is this static?

Reply:
It's a little more efficient not to resize the vectors with every iteration

Reply:
Cool! Did not know about this.